e2e/web/src/web-vite.test.ts

```diff
@@ -60,7 +60,7 @@ describe('Web Components Applications with bundler set as vite', () => {
 
     runCLI(`generate @nrwl/web:app ${appName} --bundler=vite --no-interactive`);
     runCLI(
-      `generate @nrwl/react:lib ${libName} --bundler=vite --no-interactive`
+      `generate @nrwl/react:lib ${libName} --bundler=vite --no-interactive --unitTestRunner=jest`
```

Comment:
is there a reason this shouldn't be vitest instead of jest?

Reply:
It was the default from before, but I can change it.